src/Halogen/ECharts.purs

```diff
@@ -78,19 +80,20 @@ echarts
   . (MonadAff (EChartsEffects eff) g)
   ⇒ H.Component HH.HTML EChartsQuery (Dimensions /\ Unit) EChartsMessage g
 echarts =
```

Comment:
But this default receiver is just `H.action $ SetDimension`. I don't mind exporting this, albeit it's kinda exported by `Query(..)`